requirements.txt

```diff
@@ -1,9 +1,9 @@
-Django==2.2.13
+Django==3.1
 django-countries==6.0
 django-cookie-law==2.0.3
 django-impersonate==1.5
```

Comment:
Django Impersonate doesn't seem to properly support Django 3.1 yet and I am getting a bunch of deprecation warnings when running tests:
```
Projects/github.com/JacobsAlumni/MemberManagement/venv/lib/python3.8/site-packages/impersonate/urls.py:15: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url(r'^search/$',
```

Reply:
Version 1.6 of `django-impersonate` does seem to fix these, but it is not out yet.